packages/playground/data-liberation/src/import/WP_Stream_Importer.php

```diff
@@ -202,19 +207,34 @@ public function set_frontloading_retries_iterator( $frontloading_retries_iterato
 	 */
 	private $importer;
 
-	public function next_step() {
+	/**
+	 * Calculate next steps in the import process.
+	 *
+	 * @param int $count The number of entities to process in one go.
+	 *
+	 * @return bool
+	 */
+	public function next_step( $count = 10000 ) {
```

Comment:
`$count` as an argument is misleading as it only applies to two stages but not the other ones. The only reason to customize this I can think of are unit tests – perhaps let's turn this into a constant until we need to customize it, and then refactor it to a class property?